pom.xml

```diff
@@ -47,12 +47,12 @@
             <dependency>
                 <groupId>javax.xml.bind</groupId>
                 <artifactId>jaxb-api</artifactId>
-                <version>2.3.0</version>
+                <version>2.4.0-b180830.0359</version>
```

Comment:
They have a weird release system. Even though it's in maven central, this appears to be a pre-release candidate. I did a bit of detective work [here](https://github.com/javaee/jaxb-v2/releases).

Reply:
The latest stable version is ```2.3.1```, but in the maven repo the latest stable available is ```2.3.0```. I have updated the same version.